which-key-posframe.el

```diff
@@ -102,7 +107,9 @@ of the buffer text to be displayed in the popup"
 
 (defun which-key-posframe--hide ()
   "Hide which-key buffer when posframe popup is used."
-  (when (buffer-live-p which-key--buffer)
+  (when (and (buffer-live-p which-key--buffer)
+             which-key-posframe--popup-visible)
```

Comment:
Not there's any reason to keep the `buffer-live-p` call anymore, and it seems to work without it, so maybe I should remove it?

Alternatively, is there another way to check if the popup is visible, so that we don't need `which-key-posframe--popup-visible'?

Reply:
relevant: https://github.com/fejfighter/which-key-posframe/commit/5eb8a72815dfd8358e81f3fbcbb2d65e55ce8949